development/webpack/webpack.desktop.config.js

```
@@ -16,12 +16,22 @@ module.exports = ({
   const configs = ENABLE_ANALYZER
     ? [analyzerConfig({ configName: platform })]
     : [];
+
+  // Renderer process externals - only exclude packages that shouldn't be bundled
+  const commonDesktopConfig = {
+    externals: {
+      // Exclude the entire BLE transport package to prevent Node.js modules from leaking to renderer
+      '@onekeyfe/hd-transport-electron': 'commonjs @onekeyfe/hd-transport-electron',
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)